frontend/my-adopthive-app/src/App/App.jsx

```diff
@@ -32,6 +34,8 @@ function App() {
           <Route path="/adopter-list" element={user && role === 'Adoptee' ? <AdopterList adopteeId={user.id} /> : <Navigate to="/signin" />} />
           <Route path="/adopter-home" element={user && role === 'Adopter' ? <AdopterHome /> : <Navigate to="/signin" />} />
           <Route path="/adoptee-home" element={user && role === 'Adoptee' ? <AdopteeHome /> : <Navigate to="/signin" />} />
+          <Route path="/adoptee-favourites" element={user && role === 'Adoptee' ? <AdopteeFavourites adopteeId={user.id} /> : <Navigate to="/signin" />} />
+          <Route path="/adopter-favourites" element={user && role === 'Adopter' ? <AdopterFavourites adopterId={user.id} /> : <Navigate to="/signin" />} />
```

Comment:
great use of single responsibility principle! I like the use of separate components to keep things organized and well-contained.